packages/esm-service-queues-app/src/overlay.test.tsx

```diff
@@ -1,51 +0,0 @@
-import React from 'react';
```

Comment:
Looks like this test didn't make it into Core. It'd be nice if it did.

Reply:
Core PR coming soon